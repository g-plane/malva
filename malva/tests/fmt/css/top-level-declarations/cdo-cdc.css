--accent: var(--<!-- TEMPLATE-SLOT-ACCENT-COLOR -->)
