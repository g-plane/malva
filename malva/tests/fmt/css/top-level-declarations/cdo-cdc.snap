---
source: malva/tests/fmt.rs
---
--accent: var(--<!-- TEMPLATE-SLOT-ACCENT-COLOR -->);
